org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/embedded/EmbeddedEditorFactory.java

```diff
@@ -139,6 +139,16 @@
 		protected Boolean readonly;
 		protected boolean editorBuild;
 		protected IValidationIssueProcessor issueProcessor;
+		protected int style = SWT.BORDER | SWT.V_SCROLL | SWT.H_SCROLL;
+		
+		/**
+		 * Configures the SWT style of the embedded editor.
+		 * @since 2.13
+		 */
+		public Builder style(int style){
```

Comment:
I think `withStyle` would be a better name. Would you rename it?
Please move the method after `withParent`, since it is the last builder method that was added.